I fixed this by uninstalling coverage.py with pip and installing it using easy_install.

